test/src/network.spec.ts

```diff
@@ -555,7 +555,7 @@ describe('network', function () {
       if (isChrome) {
         expect(failedRequest.failure()!.errorText).toBe('net::ERR_FAILED');
       } else {
-        expect(failedRequest.failure()!.errorText).toBe('NS_ERROR_FAILURE');
+        expect(failedRequest.failure()!.errorText).toBe('NS_ERROR_ABORT');
```

Comment:
Interesting, so CDP in Firefox sends a different code? I wonder if this would fail for the CDP-Firefox tests.

Reply:
The CDP implementation fails here anyway due to other issues.